### ISSUE TITLE
Builds started failing ' Could not find any version that matches com.facebook.react:react-native:0.13.+.'

### ISSUE THREAD

**Issue by vladikoff** [mvayngrib/react-native-randombytes#23]
```
* What went wrong:
Could not resolve all files for configuration ':app:releaseCompileClasspath'.
> Could not find any version that matches com.facebook.react:react-native:0.13.+.
  Versions that do not match:
      0.20.1
      0.14.0
      0.12.0
      0.55.4
  Searched in the following locations:
      file:/opt/android/sdk/extras/m2repository/com/facebook/react/react-native/maven-metadata.xml
      file:/opt/android/sdk/extras/m2repository/com/facebook/react/react-native/
      file:/opt/android/sdk/extras/google/m2repository/com/facebook/react/react-native/maven-metadata.xml
      file:/opt/android/sdk/extras/google/m2repository/com/facebook/react/react-native/
      file:/opt/android/sdk/extras/android/m2repository/com/facebook/react/react-native/maven-metadata.xml
      file:/opt/android/sdk/extras/android/m2repository/com/facebook/react/react-native/
      file:/home/circleci/.m2/repository/com/facebook/react/react-native/maven-metadata.xml
      file:/home/circleci/.m2/repository/com/facebook/react/react-native/
      file:/home/circleci/notes/native/native/node_modules/react-native/android/com/facebook/react/react-native/maven-metadata.xml
  Required by:
      project :app > project :react-native-randombytes
```

Why does this happen? Did something change in the maven config outside of this project, such that the dependency is not there anymore?

**Comment by vladikoff** [comment]
This here  https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml now resolves to 

```
<?xml version="1.0" encoding="UTF-8"?>
<metadata>
  <groupId>com.facebook.react</groupId>
  <artifactId>react-native</artifactId>
  <version>0.20.1</version>
  <versioning>
    <latest>0.20.1</latest>
    <release>0.20.1</release>
    <versions>
      <version>0.12.0</version>
      <version>0.14.0</version>
      <version>0.20.1</version>
    </versions>
    <lastUpdated>20180522170621</lastUpdated>
  </versioning>
</metadata>

```

per `20180522170621` maybe something changed there

**Comment by syreinrage** [comment]
Same issue, was working fine yesterday

`* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugApk'.
   > A problem occurred configuring project ':react-native-randombytes'.
      > Could not resolve all dependencies for configuration ':react-native-randombytes:_debugPublishCopy'.
         > Could not find any version that matches com.facebook.react:react-native:0.13.+.
           Versions that do not match:
               0.20.1
               0.19.1
               0.16.0
               0.15.0
               0.14.0
               + 3 more`

**Comment by caseylmanus** [comment]
I am also experiencing this issue on my local machine and our cloud build environment.

**Comment by vladikoff** [comment]
I switched our project to use a fix like this https://github.com/vladikoff/react-native-randombytes/commit/0f6e0836979bedd48733cb1d6583b6d179208003 might help unblock builds for y'all

**Comment by syreinrage** [comment]
@vladikoff thanks for the tip! I changed this to version '0.54.2' and this worked for me. for some reason, my project did not like '0.55.+'

**Comment by vladikoff** [comment]
> @vladikoff thanks for the tip! I changed this to version '0.54.2' and this worked for me. for some reason, my project did not like '0.55.+'

Right, safe bet is to set it to the react-native version that you have in your package.json

**Comment by mvayngrib** [comment]
fixed by @PythonicNinja and released in 3.2.0

closing for now, will reopen if the problem survives the fix
